### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by toddmotto**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+        td {
+            padding: 16px 12px 14px;
+        }
+    </style>
+</head>
+<body>
+
+<div ng-app="app" class="app">
+    <div ng-view></div>
```
Comment (ipc103) [inline index.html]: should this be `ui-view-`?

**Comment by ipc103** [comment]
This is a nice lab - is there a slightly different domain that we can use? Since they saw this domain in the prior example, breaking it out into a different domain here will help the student solidify understanding. Maybe like a twitter user page with nested views for profile, account, and feed?  The other needs are:
1. Tests - assuming protractor is the way to go here?
2. Solution branch

**Comment by toddmotto** [comment]
Not sure we can do Twitter APIs due to OAuth2. Maybe we could copy the API from our previous reddit lab, and get them to do nested views for subreddits/domains?

**Comment by ipc103** [comment]
We could do that - when I said domain, I really just meant the context that the lab is in. Rather than name the main page "Settings" and the child pages "Notifications", "Profile", and "User", ask the student to make a main page called "Home" and child pages called "Tweets", "Direct Messages", and "Notifications". I like that they're not actually fetching any data here - this lab is just about practicing displaying nested views. Make sense?

**Pull request merged**